t/op/sigsystem.t

```diff
@@ -50,8 +60,11 @@ sub test_system {
     my $got_zeroes      = 0;
 
     # This test is looking for a race between system()'s waitpid() and a
-    # signal handler.    Looping a few times increases the chances of
-    # catching the error.
+    # signal handler. The system() call is expected to not interfere with the 
+    # SIGCHLD signal handler. In particular, the wait() called within system() 
+    # is expected to reap the child process forked by system() before the 
+    # SIGCHLD signal handler is called. 
+    # Looping a few times increases the chances of # catching the error.
```

Comment:
You should remove the second '`#`' character in line 67 above.

Reply:
Will fix this typo.